user-interface/src/main/java/life/qbic/datamanager/views/projects/project/measurements/MeasurementDetailsComponent.java

```diff
@@ -242,6 +242,28 @@ private void createNGSMeasurementGrid() {
             ngsMeasurement -> asClientLocalDateTime(ngsMeasurement.registrationDate())
                 .format(DateTimeFormatter.ofPattern(DATE_TIME_FORMAT)))
         .setAutoWidth(true);
+    ngsMeasurementGrid.addComponentColumn(measurement -> {
+          if (!measurement.isPooledSampleMeasurement()) {
+            Span singularComment = new Span();
+            singularComment.setText(
+                measurement.specificMeasurementMetadata().stream().findFirst().orElseThrow().comment()
```

Comment:
I am not sure what happens here. I also think, that we should handle the `orElseThrow()` with care, since it will cause exceptions that are not of importance.